theano/sandbox/rng_mrg.py

```diff
@@ -1395,25 +1415,27 @@ def multinomial_wo_replacement(self, size=None, n=1, pvals=None, ndim=None, dtyp
 
         if size is not None:
             raise ValueError("Provided a size argument to "
-                             "MRG_RandomStreams.multinomial_wo_replacement, which does not use "
-                             "the size argument.")
+                             "MRG_RandomStreams.multinomial_wo_replacement, "
+                             "which does not use the size argument.")
         if ndim is not None:
             raise ValueError("Provided an ndim argument to "
-                             "MRG_RandomStreams.multinomial_wo_replacement, which does not use "
-                             "the ndim argument.")
+                             "MRG_RandomStreams.multinomial_wo_replacement, "
+                             "which does not use the ndim argument.")
         if pvals.ndim == 2:
             # size = [pvals.shape[0], as_tensor_variable(n)]
-            size = pvals[:,0].shape * n
+            size = pvals[:, 0].shape * n
             unis = self.uniform(size=size, ndim=1, nstreams=nstreams)
             op = multinomial.MultinomialWOReplacementFromUniform(dtype)
             n_samples = as_tensor_variable(n)
             return op(pvals, unis, n_samples)
         else:
-            raise NotImplementedError(("MRG_RandomStreams.multinomial_wo_replacement only"
-                                       " implemented for pvals.ndim = 2"))
+            raise NotImplementedError(
+                ("MRG_RandomStreams.multinomial_wo_replacement "
```

Comment:
This line seems too short. Make sure you break the line at the last position possible